frontend/server/src/main/java/org/pytorch/serve/ServerInitializer.java

```diff
@@ -65,6 +69,14 @@ public void initChannel(Channel ch) {
                                     PluginsManager.getInstance().getInferenceEndpoints()));
             httpRequestHandlerChain =
                     httpRequestHandlerChain.setNextHandler(new WorkflowInferenceRequestHandler());
+
+            // Added OIP protocol with inference connector
+            if (ConfigManager.getInstance().isOpenInferenceProtocol()) {
+                logger.info("OIP added with handler chain");
+                httpRequestHandlerChain = httpRequestHandlerChain.setNextHandler(
+                        new OpenInferenceProtocolRequestHandler());
+            }
```

Comment:
According to the implementation (frontend/server/src/main/java/org/pytorch/serve/http/api/rest/OpenInferenceProtocolRequestHandler.java), it seems that [kserver v2 inference request](https://github.com/kserve/kserve/blob/master/docs/predict-api/v2/required_api.md#inference) is unable to be processed.